app.py

```diff
@@ -0,0 +1,54 @@
+import preprocessor as pp
+import pandas as pd
+import pickle
+from sklearn.feature_extraction.text import CountVectorizer
+
+class Application:
+    def __init__(self):
+        self.source_data = []
+        self.import_model_vocabulary()
+        self.preprocessor = pp.Preprocessor()
+        self.vectorizer = CountVectorizer(vocabulary=self.vocabulary)
+
+    def import_model_vocabulary(self):
+        with open('model.pickle', 'rb') as file:
+            self.model = pickle.load(file)
+            
+        with open('vocabulary.pickle', 'rb') as file:
+            self.vocabulary = pickle.load(file)
+
+    def get_user_input(self):
+        self.source_data = []
```

Comment:
This is initialized in constructor of this class and therefore this declaration is redundant.

Reply:
It's necessary because we need to clean the array every time we get the new input.

Reply:
Actually we don't need this in constructor. I'll remove it.